### DISCUSSION TITLE
Not able to connect to Ender 3V3 KE

### DISCUSSION THREAD

**Discussion by simonsays10** [OrcaSlicer/OrcaSlicer#5545] [General]
I downloaded and installed the new OrcaSlicer 2.1.0 and ever since I did that, it's not connecting to my printer.  I'm not sure why.  When I go to the pre-settings in orca for the printer, the wifi is correct. I put :4408 and :4409, and deleted, and all 3 settings still won't connect to printer.  Keeps saying bad gateway.  I've run the test several times with all those configurations and it keeps saying bad gateway. When I go to device in orca, and click reconnect or force refresh it says bad gateway. I'm not sure how to fix this. Can anybody help?

**Comment by sawpdx** [comment]
Did you get this resolved?
